vendor/github.com/paultyng/go-newrelic/cmd/alert_channels.go

```diff
@@ -0,0 +1,57 @@
+package cmd
+
+import (
+	"github.com/imdario/mergo"
+	"github.com/paultyng/go-newrelic/api"
+	"github.com/spf13/cobra"
+)
+
+func makeChannelsCmd(dst cobra.Command) *cobra.Command {
```

Comment:
not sure what is up with the vendoring on this PR as well, I don't think it should be adding this vendor stuff in on the PR since its not related to your code change.

Reply:
It might have to do with how I added it to the govendor directory, I needed the latest of go-newrelic with my key transaction changes, so I just updated the version of that. ListKeyTransactions did not exist in the older version. Not sure if I should have done that differently?

Reply:
I'm not super familiar with `govendor`, still ramping up on it a little, did you use `govendor update` or do it in some other way?

Reply:
To be clear there will be some vendor changes, there are just a lot of other files in here as well.

Reply:
Same here, I come from a glide/dep world. Iirc I manually updated it at first, and then as it failed the govendor checks I went back and govendor add +missing github.com/paultyng/go-newrelic, then did a govendor status to make sure things were ok.

Reply:
Sorry I missed your second comment. You're talking about the /cmd package right? Its probably because of my manual update of the repo initially, I'll have to look at it to see how to remove that in the proper govendor way

Reply:
Yeah, I wish I had a better answer on the exact commands to fix it, but seems like you are on the right track.

Reply:
Just went through the process for my PR (#26), you'll want to run this to get the latest from the `go-newrelic` library:

```
govendor fetch github.com/paultyng/go-newrelic/api
```

the `/api` bit was important, otherwise it pulls in a lot of other libraries (you'll see some a couple new files in the `vendor/github.com/paultyng/go-newrelic/api/` directory and a new `vendor.json` but that's it). 

If the `fetch` works, be sure to verify it builds and govendor isn't complaining by running `govendor status` (exit code 0) otherwise the TravisCI build will fail.

Reply:
@smithclay I'm not sure why but running that above command pulls down everything in /cmd

Reply:
@smithclay ah nvm me I did the wrong thing, I should be able to have this fixed up soon